src/main/java/frc/robot/swerve/ModuleIOSim.java

```diff
@@ -38,43 +39,45 @@ public ModuleIOSim() {
                 1 / SwerveConstants.ANGLE_REDUCTION,
                 SwerveConstants.ANGLE_MOTOR_MOMENT_OF_INERTIA);
 
-        angleFeedback = new PIDController(3.5, 0, 0, 0.02);
-        velocityFeedback = new PIDController(0.5, 0, 0.00, 0.02);
+        angleFeedback = new PIDController(8, 0, 0, 0.02);
+        velocityFeedback = new PIDController(3.5, 0, 0.00, 0.02);
```

Comment:
1. "controller" is a better term.
2. You should use the same gains used for the real implementation.

Reply:
1. I agree
2. The gains can't be the same as the real robot, there are unit conversions and behind the scenes of the TalonFX, and real world factors.

Reply:
e3534ac653ea1fcd161ceecca3e6524b87ebc43c